src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -42,12 +42,12 @@ public void testAddUsers() {
         ctl.addUser(
             ds, /*userID=*/ 1l, /*AuthID=*/ 2l,
             /*userName=*/ "Sarah", /*userYear=*/ 2022,
-            /*userMajors=*/ user1Majors, /*skills=*/ Skills.LEADERSHIP,
+            /*userMajors=*/ user1Majors, /*skills=*/ Skills.LEADERSHIP.name(),
             /*userTotalCompTasks=*/ 3);
         ctl.addUser(
             ds, /*userID=*/ 2l, /*AuthID=*/ 3l,
             /*userName=*/ "Joe", /*userYear=*/ 2024,
-            /*userMajors=*/ user2Majors, /*skills=*/ Skills.ORGANIZATION,
+            /*userMajors=*/ user2Majors, /*skills=*/ Skills.ORGANIZATION.name(),
```

Comment:
[ERROR] /home/talsternberg/project-management-tool/src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java:[50,78] incompatible types: java.lang
.String cannot be converted to com.rtb.projectmanagementtool.user.Skills